### PR TITLE
Fix watch() function

### PR DESCRIPTION
**Pull request by lyrixx**
- Fix watch() function
- Add castor task to build phar

### CODE REVIEW

File: tests/Examples/ListTest.php.output.txt [hunk 1]
```diff
@@ -6,6 +6,14 @@ no-namespace                  Task without a namespace
 args:another-args             Dumps all arguments and options, without configuration
 args:args                     Dumps all arguments and options, with custom configuration
 bar:bar                       Prints bar, but also executes foo
+castor:phar:build             Build watcher for all systems
+castor:phar:darwin            Build watcher for MacOS system
+castor:phar:linux             Build watcher for Linux system
+castor:phar:windows           Build watcher for Windows system
```
Comment (pyrech) [inline tests/Examples/ListTest.php.output.txt:12]:
```suggestion
castor:phar:build             Build phar for all systems
castor:phar:darwin            Build phar for MacOS system
castor:phar:linux             Build phar for Linux system
castor:phar:windows           Build phar for Windows system
```

File: tools/phar/castor/build.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace castor\phar;
+
+use Castor\Attribute\AsTask;
+
+use function Castor\parallel;
+use function Castor\run;
+
+#[AsTask(description: 'Build watcher for Linux system')]
```
Comment (pyrech) [inline tools/phar/castor/build.php:10]:
```suggestion
#[AsTask(description: 'Build phar for Linux system')]
```

File: tools/phar/castor/build.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+function linux()
+{
+    run('vendor/bin/box compile -c box.linux-amd64.json', path: __DIR__ . '/..');
+}
+
+#[AsTask(description: 'Build watcher for MacOS system')]
```
Comment (pyrech) [inline tools/phar/castor/build.php:16]:
```suggestion
#[AsTask(description: 'Build phar for MacOS system')]
```

File: tools/phar/castor/build.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+function darwin()
+{
+    run('vendor/bin/box compile -c box.darwin-amd64.json', path: __DIR__ . '/..');
+}
+
+#[AsTask(description: 'Build watcher for Windows system')]
```
Comment (pyrech) [inline tools/phar/castor/build.php:22]:
```suggestion
#[AsTask(description: 'Build phar for Windows system')]
```

File: tools/phar/castor/build.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+function windows()
+{
+    run('vendor/bin/box compile -c box.windows-amd64.json', path: __DIR__ . '/..');
+}
+
+#[AsTask(description: 'Build watcher for all systems')]
```
Comment (pyrech) [inline tools/phar/castor/build.php:28]:
```suggestion
#[AsTask(description: 'Build phar for all systems')]
```

**Review by pyrech — CHANGES_REQUESTED** [review]

**Comment by lyrixx** [comment]
@pyrech thanks for the review 👍🏼 I fixed the code

**Review by pyrech — APPROVED** [review]

**Pull request merged**